website/data/plugin-testing-nav-data.json

```diff
@@ -0,0 +1,150 @@
+[
```

Comment:
I think we can either exclude this file or leave it empty for now 👍

Reply:
Refer also to https://github.com/hashicorp/terraform-plugin-testing/pull/34